### ISSUE TITLE
Add release team

### ISSUE THREAD

**Issue by Kansea** [ros2-gbp/ros2-gbp-github-org#314]
* Name of the new release team: techmagic
* Release repositories to add:
  
  * [weight_scale_interfaces](https://github.com/TechMagicKK/weight_scale_interfaces)
      * [ros/rosdistro source entry](https://github.com/ros/rosdistro/blob/f79a517a60de00fae0439bf7097165ea54fa4f04/humble/distribution.yaml#L8710C9-L8710C9)
        * [x] There is an existing release repository which should be imported: <RELEASE REPOSITORY URL>
  * [aandd_ekew_driver_py](https://github.com/TechMagicKK/aandd_ekew_driver_py)
      * [ros/rosdistro source entry]()
        * [x] There is an existing release repository which should be imported: <RELEASE REPOSITORY URL>

* Team members
  * @Kansea
  * [@longjie0723](https://github.com/TechMagicKK/weight_scale_interfaces/blob/main/package.xml)
  * @s000ne
  * @XiandiShan

**Comment by nuclearsandwich** [comment]
Hi @Kansea, the ros/rosdistro source entry link appears to link to an existing release repository rather than a reference to the `weight_scale_interfaces` repository listed in https://github.com/ros/rosdistro. For example, here is `rclcpp`'s source entry: https://github.com/ros/rosdistro/blob/e08b9f1b4b511dcbf247d08e2ff5b8ef1e65bdfd/rolling/distribution.yaml#L4114-L4118

Once that PR is opened and merged, we can proceed with creating the release repository.

**Comment by Kansea** [comment]
Hi @nuclearsandwich, i have already updated the source entry link in the PR https://github.com/ros/rosdistro/pull/38254.
Do I need to wait for the approval of the administrator now?

**Comment by Kansea** [comment]
Hi @nuclearsandwich, i have already modified version "humble" to "main" in source entry in the PR https://github.com/ros/rosdistro/pull/38254 yesterday. 
 Could you please help me execute the workflow again.

**Comment by nuclearsandwich** [comment]
The new package needs to undergo REP-144 naming and license review I've requested that review in https://github.com/ros/rosdistro/pull/38254#issuecomment-1694102242 once it is complete, we can proceed with the release repository setup.

**Comment by Kansea** [comment]
Hi @nuclearsandwich , we have opened a new PR [ros/rosdistro#38416](https://github.com/ros/rosdistro/pull/38416) to release a new package named "aandd_ekew_driver_py". Thank you

**Comment by nuclearsandwich** [comment]
The updates here are deploying now and I have mirrored your existing release repositories into their ros2-gbp counterparts. I recommend that you do the following to complete the migration into ros2-gbp:

1. Archive the repositories https://github.com/TechMagicKK/aandd_ekew_driver_py-release and https://github.com/TechMagicKK/weight_scale_interfaces-release so that they are not pushed to unintentionally.
2. Review the following PR updating the release repository URLs for Humble https://github.com/ros/rosdistro/pull/38559
